src/components/user/LoginForm.js

```diff
@@ -17,37 +17,54 @@ const messages = defineMessages({
   password: { id: 'login.form.password' }
 });
 
-export const LoginForm = ({ handleSubmit, error, submitting, intl }) => (
-  <form onSubmit={handleSubmit}>
-    {error && <strong>{error}</strong>}
-    <div>
-      <Field
-        name="email"
-        label={intl.formatMessage(messages.email)}
-        component={Input}
-        type="email"
-      />
-    </div>
+const DisplayErrors = ({ error }) => {
```

Comment:
This prop should be called errors since you are using a map function over it, right?